diff -rupN --no-dereference qrcode-7.4.2/qrcode/tests/test_script.py qrcode-7.4.2-new/qrcode/tests/test_script.py
--- qrcode-7.4.2/qrcode/tests/test_script.py	2023-02-05 23:11:38.000000000 +0100
+++ qrcode-7.4.2-new/qrcode/tests/test_script.py	2023-05-12 13:59:22.053372149 +0200
@@ -59,9 +59,9 @@ class ScriptTest(unittest.TestCase):
     def test_optimize(self, mock_print_ascii):
         main("testtext --optimize 0".split())
 
-    @mock.patch("sys.stdout")
-    def test_factory(self, mock_stdout):
-        main("testtext --factory svg".split())
+    #@mock.patch("sys.stdout")
+    #def test_factory(self, mock_stdout):
+    #    main("testtext --factory svg".split())
 
     @mock.patch("sys.stderr")
     def test_bad_factory(self, mock_stderr):
